File: documentation/user/en/query/ordering/examples/price/price-natural.java
final EvitaResponse<SealedEntity> entities = evita.queryCatalog(
	"evita",
	session -> {
		return session.querySealedEntity(
			query(
				collection("Product"),
				filterBy(
					priceInPriceLists("basic"),
					priceInCurrency(Currency.getInstance("EUR"))
				),
				orderBy(
					priceNatural(DESC)
				),
				require(
					entityFetch(
						priceContentRespectingFilter()
					)
				)
			)
		);
	}
);

File: documentation/user/en/query/ordering/examples/price/price-natural.cs
EvitaResponse<SealedEntity> entities = evita.QueryCatalog(
	"evita",
	session => session.QuerySealedEntity(
        Query(
        	Collection("Product"),
        	FilterBy(
        		PriceInPriceLists("basic"),
        		PriceInCurrency(new Currency("EUR"))
        	),
        	OrderBy(
        		PriceNatural(Desc)
        	),
        	Require(
        		EntityFetch(
        			PriceContentRespectingFilter()
        		)
        	)
        )
	)
);